.github/workflows/validate.yml

```diff
@@ -6,7 +6,7 @@ on:
   release:
     types:
       - created
-name: Validate
+name: validate
```

Comment:
The mixed case here is showing up in the GitHub UI. I'd like to change it to all lowercase for consistency with the file name and it's more common practice to use all lowercase for build tasks.